### PR TITLE
Refactor/storybook

### PR DESCRIPTION
**Pull request by nathanbowang**

### CODE REVIEW

File: components/Header/SecondaryNav/index.stories.tsx [hunk 1]
```diff
@@ -1,54 +1,33 @@
...
+          <MockBlock id="section-three">Section 3</MockBlock>
+          <MockBlock id="section-four">Section 4</MockBlock>
         </>
       )
     },
   ],
   parameters: {
     a11y: {
-      element: '#root header ~ *:not(main)',
+      element: '#root *:not(.mockBlock, .mockBlock *)',
```
Comment (reubenberghan) [inline components/Header/SecondaryNav/index.stories.tsx]: I think the `:not()` pseudo-class is not working here. Should probably look like the following if we are wanting to exclude those `MockBlock` components :grin:
```
element: '#root :not([class*="mockBlock"])'
```

File: components/Header/index.stories.tsx [hunk 2]
```diff
@@ -9,13 +9,16 @@ const meta: Meta = {
...
+        <MockBlock>Section 2</MockBlock>
+        <MockBlock>Section 3</MockBlock>
+        <MockBlock>Section 4</MockBlock>
       </>
     ),
   ],
   parameters: {
     a11y: {
-      element: '#root *:not(main, main *)',
+      element: '#root *:not(.mockBlock, .mockBlock *)',
```
Comment (reubenberghan) [inline components/Header/index.stories.tsx]: Same as the other stories file with regard to the `:not()` pseudo-class 😁

**Review by reubenberghan — CHANGES_REQUESTED** [review]
Nice, just needs a tweak to the a11y parameters for the stories though.

File: components/Header/index.stories.tsx (hunk 2 above)
Comment (nathanbowang) [reply components/Header/index.stories.tsx]: Cool. Thanks for the trick to select a class name with a wildcard.

File: components/Header/index.stories.tsx [hunk 3]
```diff
@@ -18,7 +19,7 @@ const meta: Meta = {
   ],
   parameters: {
     a11y: {
-      element: '#root :not([class*="mockBlock"]):not([class*="mockBlock"] *)',
+      element: `#root :not(.${style.mockBlock}):not(.${style.mockBlock} *)`,
```
Comment (nathanbowang) [inline components/Header/index.stories.tsx:22]: Ha, after a second thought. I took another approach, so it won't depend on css loader's naming configuration: 
```js
loader.options.modules = {
  auto: true,
  localIdentName: '[name]_[local]__[hash:base64:5]',
}
```
The default name is just a hash string, and `class*='mockBlock'` won't match.

Comment (reubenberghan) [reply components/Header/index.stories.tsx:22]: That config just adds a hash to the end. It looks like `mockBlock` (or whatever class name is in the css file) is the `local` variable in that config so `class*="mockBlock"` will work. However happy with your approach as that is going to use the full class.

image

**Review by reubenberghan — APPROVED** [review]

**Pull request merged**